src/components/GetDetails.js

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from 'react';
+
+export default function GetData() {
+  const [data, setData] = useState(null);
+
+  useEffect(() => {
+    const storedData = localStorage.getItem('data');
+    if (storedData) {
+      setData(JSON.parse(storedData));
```

Comment:
Is this the point that you're saying its not working? A few things I would check:

1. Is `storedData` defined? Ie. does anything come up when you add `console.log(storedData)`
2. `rating` could be a string so you may need to do `Number(data.rating)`

A separate point, the key that you're storing the data under `'data'` I would put that into a 'constant' eg. `export const LOCAL_STORAGE_KEY="data"` and use that at any points where you're setting or getting the data

Reply:
Hey @Jbarget 

_Is storedData defined? Ie. does anything come up when you add console.log(storedData)_
Storedata is defined as a variable within the useEffect hook. It is storing everything else on the page for example the title, artist, location, description and time. However it is not storing the numeric value of the star rating. It was storing a numeric value when it was just an input box which you could fill with a number between 1 and 5. 

Thank you for pointing out the folowing, hadn't come to our attention:

_A separate point, the key that you're storing the data under 'data' I would put that into a 'constant' eg. export const LOCAL_STORAGE_KEY="data" and use that at any points where you're setting or getting the data_

Reply:
So what value is being stored for `rating`? Is there any value at all for it?